HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -33,6 +36,8 @@ public ExaminationController(IExaminationService examinationService)
         /// </summary>
         /// <param name="patientJmbg">patients jmbg</param>
         /// <returns>if alright returns code 200(Ok), if not 404(not found)</returns>
+        /// 
+        [Authorize(Roles = UserRoles.Patient)]
         [HttpGet("by-patient/{patientJmbg}")]
         public ActionResult GetExaminationsByPatient(string patientJmbg)
         {
```

Comment:
I think this endpoint should be transformed to get the jmbg from the Principal like GetPreviousExaminationsByPatient does.